nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -68,8 +72,10 @@
         + "To retrieve more records, use the ScrollElasticsearchHttp processor.")
 @WritesAttributes({
         @WritesAttribute(attribute = "filename", description = "The filename attribute is set to the document identifier"),
+        @WritesAttribute(attribute = "es.hitCount", description = "The number of hits for a query"),
```

Comment:
I believe, in general, it is a standard practice in the project to avoid casing `WritesAttributes` so shall we change it to `es.hit.count` or `es.hitcount` ?

Reply:
done

Reply:
Just found out that some processors have used camel cases in writing attributes. Sorry about that. Feel free to revert back although `es.hitcount` itself looks good to me.

Reply:
nah, it is fine by me